Nos/Views/NoteComposer/ComposerActionBar.swift

```diff
@@ -135,6 +85,84 @@ struct ComposerActionBar: View {
         )
     }
     
+    var defaultMenu: some View {
+        HStack(spacing: 0) {
+            if !showPreview {
+                // Attach Media
+                ImagePickerButton { imageURL in
+                    Task {
+                        await uploadImage(at: imageURL)
+                    }
+                } label: {
+                    Image.attachMediaButton
+                        .foregroundColor(.secondaryTxt)
+                        .frame(minWidth: 44, minHeight: 44)
+                }
+                .padding(.leading, 8)
+                .accessibilityLabel(Text(.localizable.attachMedia))
+                
+                // Expiration Time
+                if let expirationTime, let option = ExpirationTimeOption(rawValue: expirationTime) {
+                    ExpirationTimeButton(
+                        model: option,
+                        showClearButton: true,
+                        isSelected: Binding(get: {
+                            self.expirationTime == option.timeInterval
+                        }, set: {
+                            self.expirationTime = $0 ? option.timeInterval : nil
+                        })
+                    )
+                    .accessibilityLabel(Text(.localizable.expirationDate))
+                    .padding(12)
+                } else {
+                    Button {
+                        subMenu = .expirationDate
+                    } label: {
+                        Image.disappearingMessages
+                            .foregroundColor(.secondaryTxt)
+                            .frame(minWidth: 44, minHeight: 44)
+                    }
+                }
+            }
+
+            Spacer() 
+
+            Text(.localizable.preview)
```

Comment:
A comment that just says "Preview Toggle" or something would be nice since we have comments for the other main elements in the action bar.

Reply:
Agreed. An alternative would be to define a `var previewToggle: some View` and do the same for the others.

Reply:
I followed @joshuatbrown's suggestion and defined `vars` for every `View` inside `defaultMenu`.